alpaca_trade_api/rest.py

```diff
@@ -282,9 +282,9 @@ def submit_order(self,
             'time_in_force': time_in_force,
         }
         if limit_price is not None:
-            params['limit_price'] = limit_price
+            params['limit_price'] = FLOAT(limit_price)
         if stop_price is not None:
-            params['stop_price'] = stop_price
+            params['stop_price'] = FLOAT(stop_price)
```

Comment:
Do you want to keep it consistent in the bracket legs as well?

Reply:
done